dockers/docker-base-stretch/Dockerfile.j2

```diff
@@ -50,12 +50,8 @@ RUN apt-get update &&        \
 # Install dependencies of supervisor
         python-pkg-resources \
         python-meld3         \
-# dependencies of redis-tools
-        libatomic1           \
-        libjemalloc1         \
-        liblua5.1-0          \
-        lua-bitop            \
-        lua-cjson            \
+# Install redis-tools
+        redis-tools=5:5.0.3-3~bpo9+2     \
```

Comment:
PR title should be updated, as this PR also updates Redis version in Stretch.

Reply:
No upgrade in Stretch, but we use vanilla instead of build. Do you have proposed title even though?

---
In reply to: [446458340](https://github.com/Azure/sonic-buildimage/pull/4732#discussion_r446458340) [](ancestors = 446458340)

Reply:
Maybe, "[Redis] Install vanilla packages for Buster and Stretch; upgrade Buster to 6.0.5"?